lib/data_update_scripts/20210712162220_backfill_section_column_for_navigation_links.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class BackfillSectionColumnForNavigationLinks
+    OTHER_LINKS = [
+      "/code-of-conduct",
+      "/privacy",
+      "/terms",
+    ].freeze
+
+    def run
+      NavigationLink.where(url: OTHER_LINKS).find_each do |link|
+        link.update(section: "other")
+      end
```

Comment:
We know there are only 3 links to update so I think this is good enough:

```suggestion
      NavigationLink.where(url: OTHER_LINKS).update(section: :other)
```